tests/numpy/test_arithmetic.py

```diff
@@ -126,9 +144,48 @@ def test_inner_outer(nps_app_inst):
     assert np.allclose(np.inner(A, B), nps.inner(nps_A, nps_B).get())
     assert np.allclose(np.outer(A, B), nps.outer(nps_A, nps_B).get())
 
+def test_broadcast(nps_app_inst):
+    import numpy as np
+    from nums import numpy as nps
+    import pytest
+    assert nps_app_inst is not None
+
+    def check_matrix_broadcast_mismatch_simple(_np_a, _np_b):
+        _ops = ['add', 'subtract']
+
+        for _op in _ops:
+            ns_op = nps.__getattribute__(_op)
+            np_op = np.__getattribute__(_op)
+            _ns_a = nps.array(_np_a)
+            _ns_b = nps.array(_np_b)
+
+            with pytest.raises(ValueError):
+                np_op(_np_a, _np_b)
+            with pytest.raises(ValueError):
+                # TODO(bcp): Add synchronous broadcast checks for simple operands.
```

Comment:
Add a space between TODO and your initials, so the TODO reads `# TODO (bcp): ...`